app/http.php

```diff
@@ -278,10 +279,11 @@ function removeAllRuntimes(Table $activeRuntimes, Pool $orchestrationPool): void
     ->param('remove', false, new Boolean(), 'Remove a runtime after execution.', true)
     ->param('cpus', 1, new Integer(), 'Container CPU.', true)
     ->param('memory', 512, new Integer(), 'Comtainer RAM memory.', true)
+    ->param('version', 'v3', new WhiteList(['v2', 'v3']), 'Runtime Open Runtime version.', true)
```

Comment:
May be a good idea to declare these as constants somewhere

Reply:
what's your opinion on using a different endpoint for v3 runtimes ? currently the endpoint is `v1/runtimes`. we could also use `/v2/runtimes` .

Reply:
I feel like that would introduce a lot of code duplication. Most of the code stays the same, just the way request is built is different (and how response is parsed).

But if you see a benefit in it, we can surely do it.